res/values-ar/strings.xml

```diff
@@ -869,7 +869,7 @@
     <string name="TEAM_DEATHMATCH">"فِرق الموت"</string>
     <string name="Speed_Click_Toggle">"تبديل سرعة الضغط"</string>
     <string name="MOD">"مُدير"</string>
-    <string name="FAQ">الأسئلة الأكثر طرحاً</string>
+    <string name="FAQ">الأسئلة الأكثر اطرحاً</string>
```

Comment:
The old translation is correct. He made a wrong edit.
```suggestion
    <string name="FAQ">الأسئلة الأكثر اطرحاً</string>
```
```suggestion
    <string name="FAQ">الأسئلة الأكثر اطرحاً</string>
```

Reply:
> 😆I think you are trying hard to get revenge
> Little child, go find a suitable job for you

- I only want to find the best translation for the game.
-  The Edit  you made is completely wrong.
- The old translation is the correct one. 
- I am not your friend to joke with.

Reply:
@mody515 can you please explain with fair details why the previous text was correct?

Reply:
Ma'am, I think he's just trying to frustrate my translation. Because the day before, I criticized his translation because it was wrong. Don't pay attention to him, he's just trying to take revenge or something 🤦🏻
Ty

Reply:
I noticed it, Taki. I just want to see if he has their reasons enough for that. ^^

Reply:
@Jeii20,I think we agree that nothing is more correct than dictionaries.
In the following pictures, I will explain the meaning of the word (طرح), and its derivatives, one of which is (طرحاً), and in the dictionary we will never find the word (اطرحاً), or at least the word (اطرحاً) has no connection to the word FAQ
![C1A59457-4D1E-4EE9-9719-A9ABF264073D](https://github.com/user-attachments/assets/0e23f060-6961-414e-87df-157b929ca91d)
Here also from the dictionary is the translation of the word (FAQ) from English to Arabic.
![37C31838-1D9A-49DF-AB16-F81CB5CF8032](https://github.com/user-attachments/assets/84e9a6e1-b046-45d7-8d5d-2c523f22291c)
![A9A05EF0-3E20-40CB-9788-D051F6986F46](https://github.com/user-attachments/assets/18d3c555-884a-4165-89ec-2b6ec572de29)

Reply:
Thanks for the clarification . Is there anything else? 
Now I'm glad you corrected this simple mistake. But there are correct translations that there is no need to criticize because they are indeed correct

Reply:
> Thanks for the clarification . Is there anything else?
> I fixed it ♥ty

You could have fixed it from the beginning instead of getting into arguments, we are looking for the best.... In the end, thank you for your understanding and correcting the mistake. Have a nice day😊🤍.